core/spec/models/spree/order/payment_spec.rb

```diff
@@ -185,15 +213,17 @@ module Spree
         context "for canceled orders" do
           before { order.update_attributes(state: 'canceled') }
 
-          it "it should be a negative amount incorporating reimbursements" do
+          it "it should be a zero amount incorporating reimbursements" do
```

Comment:
this specs is saying that it should be a zero amount but you expect -10.  Probably the description is wrong?